COC_PyBay_Reporting.md

```diff
@@ -0,0 +1,38 @@
+#Code of Conduct Reporting Guide
+
+In any PyBay space (events or online), if you believe someone is violating the Code of Conduct, we ask that you report it in one of the following ways:
+
+ - Emailing conduct@pybay.com.
+ - If there is a concern about a staff member, please reach out to one of the conference chairs, Slava (slavabal@gmail.com) or Hemanti (hemantisd@gmail.com).
+
+Reports will not be available to all staff, instead, they will be available to a limited subset of staff referred to as the Code of Conduct committee.
+
+All reports will be kept confidential where legally possible. In some cases, we may determine that a public statement will need to be made. If that’s the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise. In the event of involvement with law enforcement, we will comply with their requests as legally required.
+
+If you believe anyone is in physical danger, please notify appropriate law enforcement first. In the United States, emergency services can be contacted by dialing 911. If you are unsure which law enforcement agency is appropriate, please include this in your report and we will attempt to notify them.
+
+###WHAT TO INCLUDE IN THE REPORT
+
+ - Your contact information (so we can get in touch with you if we need to follow up)
+ - Names of any individuals involved (real names, nicknames, or pseudonyms). If there were other witnesses besides you, please try to include them as well.
+ - When and where the incident occurred. Please be as specific as possible.
+ - Your account of what occurred. If there is a publicly available record (e.g., a mailing list archive or a public chat logger) please include a link.
+ - Any extra context you believe existed for the incident.
+ - Whether you believe this incident is ongoing.
+ - Any other information you believe we should have.
+
+###WHAT HAPPENS AFTER YOU FILE A REPORT
+
+The PyBay community team aims to follow the process outlined in the Code of Conduct Response Playbook.
+
+We promise to acknowledge receipt of any reports within 24 hours or sooner, and a resolution (or update in case a resolution is delayed) within 7 days.
+
+Note: If the violation is determined to be an ongoing incident or a threat to physical safety, the staff’s immediate priority will be to protect everyone involved. This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
+
+Once we determine the final needed action, we’ll contact the original reporter to let them know what action we’ll be taking, if any. Please understand that while we take into account feedback from the reporter on the appropriateness of our response, we cannot guarantee we can or will act on it.
+
+Finally, the staff may choose to make a public report of the incident if it was reported outside one of our conferences.
```

Comment:
But not otherwise?

Reply:
I think this is understood to include inside and outside of conference behavior.